docs/hacking-guide/en/history.md

```diff
@@ -0,0 +1,19 @@
+History
+=======
+
+Artemis started with a code donation grant by RedHat in Oct 2014. During this process we took a snapshot of codebase and included it as an initial commit on the apache repository.
```

Comment:
Not my style.. I usually prefer text very concise and straight to the point.. but I can include yours.